### ISSUE TITLE
Ignore Tests not counted up

### ISSUE THREAD

**Issue by PhilippMeier** [ThrowTheSwitch/Unity#14]
function "UnityIgnoreTest" (defined in unity_fixture.c) misses to count up "Unity.TestIgnores". This then lead to the test result showing "0 Ignored" when it should show "1 Ignored".

I guessed the fix for this bug to be adding "Unity.TestIgnores++;" in function "UnityIgnoreTest()" This function would then look like this:
```c
void UnityIgnoreTest()
{
    Unity.NumberOfTests++;
    Unity.TestIgnores++;
    Unity.CurrentTestIgnored = 1;
    UNITY_OUTPUT_CHAR('!');
}
```

But after adding another TEST case (not an ignored one), the Ignore count showed 2! Then playing around a little bit showed that the problem seems to face up only if the ignored test case is the last in the group (TEST_GROUP_RUNNER)!

I now definitively know there is a bug, but my proposed solution above is not valid.

**Comment by jsalling** [comment]
This bug was fixed in commit e21881c5.
@mvandervoord Will you close this issue?

**Comment by mvandervoord** [comment]
:) Thanks!
